sdk/src/main/java/com/microsoft/hydralab/appium/ThreadParam.java

```diff
@@ -2,21 +2,26 @@
 // Licensed under the MIT License.
 package com.microsoft.hydralab.appium;
 
+import com.microsoft.hydralab.performance.PerformanceInspectionService;
+
 import java.util.Map;
 
 public class ThreadParam {
     private static InheritableThreadLocal<AppiumParam> appiumParam = new InheritableThreadLocal<>();
     private static InheritableThreadLocal<Map<String, String>> configMap = new InheritableThreadLocal<>();
+    private static InheritableThreadLocal<PerformanceInspectionService> performanceExecutor = new InheritableThreadLocal<>();
```

Comment:
We will use `ThreadLocal` to share data between ` PerformanceInspectionService` and `TestRunner`.